commands/test/test-command.ts

```diff
@@ -194,7 +242,7 @@ export const sendQuestion = async (interaction: any) => {
 
 
 
-async function conversationStarter(interaction: any, bestMatch: number[], user: number[], bestMatchUser: User) {
+async function conversationStarter(whereToSend: any, interaction: any, bestMatch: number[], user: number[], bestMatchUser: User) {
```

Comment:
Maybe instead of 'whereToSend' something more descriptive such as: 'privateChannelOfMatch'?